### ISSUE TITLE
ValueError: The server sent an unknown constructor

### ISSUE THREAD

**Issue by xFeliXx** [pyrogram/pyrogram#1263]
### Checklist

- [X] I am sure the error is coming from Pyrogram's code and not elsewhere
- [X] I have searched in the issue tracker for similar bug reports, including closed ones
- [X] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version

### Description

I'm running multiple pyrogram clients (10-30). After running for some prolonged period of time (16+ hours) sometimes the errors start to appear from within `Session.handle_packet()` coroutine, here is one example:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/session/session.py", line 187, in handle_packet
    data = await self.loop.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/crypto/mtproto.py", line 81, in unpack
    raise ValueError(f"The server sent an unknown constructor: {hex(e.args[0])}\n{left}")
ValueError: The server sent an unknown constructor: 0xf8d32aed
23baf7dc d731ed79 7339dd82 1dec3454 9ada0d9b
```
The values itself are different with every error.

Also, most of the time (but not always), before the exception there is a following info log message:
```
[7] Retrying "users.GetFullUser" due to: Request timed out
```
But it might not be related, as I sometimes get these log messages without the exceptions in question.

When the errors start to appear, they appear very frequently and won't go away until the whole app will be restarted.
As the errors are coming from the Pyrogram workers - I don't see an easy way for catching them. The only solution that comes to mind is to forcefully restart the clients from time to time.

I'm using uvloop but I'm currently running all clients within a single thread, can the errors be related to that? I saw a recommendation in the docs that it would be better to run each client inside a separate thread.

### Steps to reproduce

I don't have a certain way of how to reproduce the issue. Sometimes the errors can start to appear and sometimes not. But from what I can see they only start to appear when a 20+ clients are working at the same time, and only after the clients have been running for some time.

### Logs

```shell
Task exception was never retrieved
future: <Task finished name='Task-47502849' coro=<Session.handle_packet() done, defined at /usr/local/lib/python3.10/dist-packages/pyrogram/session/session.py:186> exception=ValueError('The server sent an unknown constructor: 0xf8d32aed\n23baf7dc d731ed79 7339dd82 1dec3454 9ada0d9b')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/crypto/mtproto.py", line 70, in unpack
    message = Message.read(data)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/message.py", line 46, in read
    return Message(TLObject.read(BytesIO(body)), msg_id, seq_no, length)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/tl_object.py", line 33, in read
    return cast(TLObject, objects[int.from_bytes(b.read(4), "little")]).read(b, *args)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/types/rpc_result.py", line 66, in read
    result = TLObject.read(b)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/tl_object.py", line 33, in read
    return cast(TLObject, objects[int.from_bytes(b.read(4), "little")]).read(b, *args)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/gzip_packed.py", line 41, in read
    return cast(GzipPacked, TLObject.read(
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/tl_object.py", line 33, in read
    return cast(TLObject, objects[int.from_bytes(b.read(4), "little")]).read(b, *args)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/types/users/user_full.py", line 77, in read
    full_user = TLObject.read(b)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/raw/core/tl_object.py", line 33, in read
    return cast(TLObject, objects[int.from_bytes(b.read(4), "little")]).read(b, *args)
KeyError: 4174588653

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/session/session.py", line 187, in handle_packet
    data = await self.loop.run_in_executor(
  File "/usr/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.10/dist-packages/pyrogram/crypto/mtproto.py", line 81, in unpack
    raise ValueError(f"The server sent an unknown constructor: {hex(e.args[0])}\n{left}")
ValueError: The server sent an unknown constructor: 0xf8d32aed
23baf7dc d731ed79 7339dd82 1dec3454 9ada0d9b
```

**Comment by delivrance** [comment]
This is a server issue, not client-related. The error will likely fade away at some point.

**Comment by xFeliXx** [comment]
@delivrance, 

I'm getting multiple different server-side errors which sooner or later fade away. But this is the only error I've encountered which doesn't fade away unless the client is restarted. I've tried to keep the clients running for a night and the error didn't fade away, but as soon as I've restarted the clients - it was gone.

So far I've only encountered this specific issue only a few times, so it's not that common.
